src/mono/mono/metadata/Makefile.am

```diff
@@ -154,9 +154,10 @@ nodist_libmonoruntime_shimglobalization_la_SOURCES = \
 									@ICU_SHIM_PATH@/pal_localeStringData.c \
 									@ICU_SHIM_PATH@/pal_normalization.c \
 									@ICU_SHIM_PATH@/pal_timeZoneInfo.c \
-									@ICU_SHIM_PATH@/pal_icushim.c
+									@ICU_SHIM_PATH@/pal_icushim.c \
+									@ICU_SHIM_QCALLS_PATH@/entrypoints.c
```

Comment:
https://github.com/dotnet/runtime/blob/master/src/coreclr/src/libraries-native/entrypoints.c Can we put this file in some shared place under `src/` 

/cc @jkotas

Reply:
`ICU_SHIM_PATH` may be a good place for this one. I agree that we need a better to share C impls between CoreCLR and Mono.

Reply:
Can I move?